### PR TITLE
Initial transformer service commit

### PR DESCRIPTION
**Pull request by tmihalac**
Skeleton Implementation of transformer service issue https://github.com/RHEcosystemAppEng/DepDraw/issues/8
Signed-off-by: Theodor Mihalache <tmihalac@redhat.com>

### CODE REVIEW

**Comment by nemerna** [comment]
@tmihalac , those should set under a new Folder on the root path called src as described in the Issue

the POC dir contains the old code that was used only for the POC of the idea of the project

File: POC/pom.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (dmartinol) [inline POC/pom.xml:1]: why in the POC filer? it was meant to store the original POC files, not the dev project files

**Review by dmartinol — APPROVED** [review]
fine for me

File: POC/pom.xml (hunk 1 above)
Comment (tmihalac) [reply POC/pom.xml]: You are right, by mistake

**Pull request merged**